commit f6003fbc8cbd6779b6e7fcf5b05293b54a4948f8
Author: Alexander E. Patrakov <patrakov@gmail.com>
Date:   Thu Nov 28 17:37:57 2019 +0500

    MDEV-21140 Make galera_recovery.sh work with fs.protected_regular = 1 (#1417)
    
    The log file is opened as root since commit bb7a70c, so there is no need
    to chown it.

diff --git a/scripts/galera_recovery.sh b/scripts/galera_recovery.sh
index 709c4b0eed5..8df2abc3fd5 100644
--- a/scripts/galera_recovery.sh
+++ b/scripts/galera_recovery.sh
@@ -101,8 +101,7 @@ wsrep_recover_position() {
 
 # Safety checks
 if [ -n "$log_file" -a -f "$log_file" ]; then
-  [ "$euid" = "0" ] && chown $user $log_file
-      chmod 600 $log_file
+  chmod 600 $log_file
 else
   log "WSREP: mktemp failed"
 fi
